db/schema.rb

```diff
@@ -86,12 +86,13 @@
     t.boolean "held", default: false
     t.boolean "hmrc_bank_validation_succeeded", default: false
     t.json "hmrc_bank_validation_responses", default: []
-    t.boolean "logged_in_with_tid", default: false
+    t.boolean "logged_in_with_tid"
```

Comment:
Why has this changed?

Reply:
Noise from hopping between branches. I will tidy.